modules/volunteers/admin/sitesupport/hourEntryField.js

```diff
@@ -0,0 +1,74 @@
+/* jshint esversion: 6 */
+/* globals   */
+
+export default {
+  props: {
+    modelValue: String
+  },
+  emits: [
+    'update:modelValue',
+    'change'
+  ],
+  data() {
+    return {
+      enterHours: null,
+      enterMin: null,
+    }
+  },
+  updated() {
+    this.enterHours = null;
+    this.enterMin = null;
+  },
+  methods: {
+    changed: function(evt) {
+      evt.target.value = parseInt(this.hours) + parseInt(this.mins) / 60.0;
+      this.$emit('update:modelValue', evt.target.value);
+      this.$emit('change');
+    },
+  },
+  computed: {
+    value: {
+      get() {
+        return this.modelValue;
+      },
+      set(value) {
+        this.$emit('update:modelValue', value);
+        this.$emit('change');
+      }
+    },
+    hours: {
+      get() {
+        if (this.enterHours != null) {
+          var v = this.enterHours;
+          this.enterHours = null;
+          return v;
+        }
+        return Math.floor(this.modelValue);
+      },
+      set(value) {
+        this.enterHours = Math.floor(value);
+      }
+    },
+    mins: {
+      get() {
+        if (this.enterMin != null) {
+          var v = this.enterMin;
```

Comment:
It looks like we're not reassigning the value at all here, so can we use `const` instead of `var` for it?